# Patch to check for GTest and Gflags in a visible way, add GTEST_BOTH_LIBRARIES for snappy-unittest
diff -urNad snappy-1.1.5-orig/CMakeLists.txt snappy-1.1.5/CMakeLists.txt
-- snappy-1.1.5-orig/CMakeLists.txt	2017-06-29 04:37:30.000000000 +0300
+++ snappy-1.1.5/CMakeLists.txt	2017-07-03 15:26:12.978826740 +0300
@@ -56,12 +56,12 @@
 CHECK_CXX_SOURCE_COMPILES("int main(void) { return __builtin_ctzll(0); }"
         HAVE_BUILTIN_CTZ)
 
-FIND_PACKAGE(GTest QUIET)
+FIND_PACKAGE(GTest)
 IF(GTEST_FOUND)
     SET(HAVE_GTEST 1)
 ENDIF()
 
-FIND_PACKAGE(Gflags QUIET)
+FIND_PACKAGE(Gflags)
 IF(GFLAGS_FOUND)
     SET(HAVE_GFLAGS 1)
 ENDIF()
@@ -146,7 +146,7 @@
 ADD_EXECUTABLE(snappy-unittest snappy_unittest.cc snappy-test.cc)
 TARGET_COMPILE_DEFINITIONS(snappy-unittest PRIVATE -DHAVE_CONFIG_H)
 TARGET_LINK_LIBRARIES(snappy-unittest snappy ${COMPRESSION_LIBS}
-                      ${GFLAGS_LIBRARIES})
+                           ${GFLAGS_LIBRARIES} ${GTEST_BOTH_LIBRARIES})
 TARGET_INCLUDE_DIRECTORIES(snappy-unittest BEFORE PRIVATE ${Snappy_SOURCE_DIR}
                            ${GTEST_INCLUDE_DIRS} ${GFLAGS_INCLUDE_DIRS})
 
